packages/exhibitor/extra/start_exhibitor.py

```diff
@@ -175,7 +175,24 @@ def invoke_detect_ip():
     print("ERROR: No known exhibitor backend:", exhibitor_backend)
     sys.exit(1)
 
+truststore_path = '/var/lib/dcos/exhibitor-tls-artifacts/truststore.jks'
+clientstore_path = '/var/lib/dcos/exhibitor-tls-artifacts/clientstore.jks'
+serverstore_path = '/var/lib/dcos/exhibitor-tls-artifacts/serverstore.jks'
+
+exhibitor_env = os.environ.copy()
+if os.path.exists(truststore_path) and \
+   os.path.exists(clientstore_path) and \
+   os.path.exists(serverstore_path):
+    exhibitor_env['EXHIBITOR_TLS_TRUSTSTORE_PATH'] = truststore_path
+    exhibitor_env['EXHIBITOR_TLS_TRUSTSTORE_PASSWORD'] = 'changeme'
```

Comment:
Please, in your entire system, change `changeme` to `not-relevant-for-security` or something shorter but comparable.